client/components/RadarChart/RadarChart.js

```diff
@@ -1,64 +1,74 @@
 import React, { Component } from 'react';
+import _ from 'lodash';
+
+// Import Functions
+import { RadarChartFunc } from '../../util/charts/RadarChart';
 
 // Import Style
 import styles from './RadarChart.scss';
 
-// Import Functions
-import { RadarChartFunc } from '../../util/charts/RadarChart';
+// Import Static Data
+import dataTalents from '../../../data/talents';
 
 export default class RadarChart extends Component {
   componentDidMount() {
-    var margin = {top: 50, right: 50, bottom: 50, left: 50},
+    let margin = {top: 50, right: 50, bottom: 50, left: 50},
     		width  = 300,
     		height = 300;
 
-    var stg = 5,
-    		ach = 4,
-    		com = 3.5,
-    		inc = 5,
-    		har = 4.75;
+    let talentsObj = _.keyBy(dataTalents, 'key');
+    let { talents } = this.props.user;
+
+    let point_1 = 5,
+    		point_3 = 4,
+    		point_5 = 3.5,
+    		point_7 = 5,
+    		point_9 = 4.75;
 
-    var baseMult = 3.75;
+    let baseMult = 3.75;
 
-    var stg_ach = ((stg+ach)/baseMult)*1.1,
-    		ach_com = (ach+com)/baseMult,
-    		com_inc = (com+inc)/baseMult,
-    		inc_har = (inc+har)/baseMult,
-    		har_stg = (har+stg)/baseMult;
+    let point_2  = (point_1+point_3)/baseMult,
+    		point_4  = (point_3+point_5)/baseMult,
+    		point_6  = (point_5+point_7)/baseMult,
+    		point_8  = (point_7+point_9)/baseMult,
+    		point_10 = (point_9+point_1)/baseMult;
 
-    var data = [
+    let data = [
     	[
-    		{axis:"stg",value:stg},
-    		{axis:"",value:stg_ach},
-    		{axis:"ach",value:ach},
-    		{axis:"",value:ach_com},
-    		{axis:"com",value:com},
-    		{axis:"",value:com_inc},
-    		{axis:"inc",value:inc},
-    		{axis:"",value:inc_har},
-    		{axis:"har",value:har},
-    		{axis:"",value:har_stg}
+    		{axis: talentsObj[talents[0]].abbreviation, value: point_1},
```

Comment:
@DimaMklk Need generate axis in loop amount of axis should releate to langth of talents